app/modules/entities/components/layouts/add_to_dot_dot_dot_menu.svelte

```diff
@@ -58,13 +59,23 @@
     }
   }
 
+  async function fetchEditions () {
+    if (!editions) {
+      editions = await getSubEntities('work', entity.uri)
+    }
+  }
+
```

Comment:
Isn't that dead code as `app/modules/entities/components/layouts/work.svelte` initializes `editions` to equal `[]` and then populates it with `getEditionsWithPublishers`?